mmdet3d/core/bbox/coders/smoke_bbox_coder.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+import torch
+
+from mmdet.core.bbox import BaseBBoxCoder
+from mmdet.core.bbox.builder import BBOX_CODERS
+
+
+@BBOX_CODERS.register_module()
+class SMOKECoder(BaseBBoxCoder):
+    """Bbox Coder for SMOKE.
+
+    Args:
+        base_depth (tuple[float]): Depth references for decode box depth.
+        base_dims (tuple[tuple[float]]): Dimension references for decode
+            box dimension for each category. fomat [l, w, h]
```

Comment:
fomat ×
Dimension references [l, w, h] for decode...

Reply:
actually is [l, h, w], thanks for pointing out.